_sass/mixins/_link-with-icon.scss

```diff
@@ -31,8 +31,10 @@
     mask: var(--link-icon) center / contain no-repeat;
   }
 
-  &:not(:hover) > img {
-    filter: #{"grayscale()"};
+  @media not (prefers-color-scheme: dark) {
+    &:not(:hover) > img {
+      filter: #{"grayscale()"};
+    }
   }
```

Comment:
**suggestion:** Actually, I believe that entire rule is unnecessary. It's an unintentional leftover from a feature that didn't get implemented.
```suggestion
```

None of the icons is coloured and needs greyscale. If we needed that, it would have to apply always, not only in hover state.

Reply:
Sounds good to me!

> If we needed that, it would have to apply always, not only in hover state.

(FWIW, it's applied when *not* hovered)